hack/test

```diff
@@ -40,6 +40,26 @@ function fmt_pass {
 	done
 }
 
+function simple_pass {
+	echo "Checking gosimple..."
+	go get honnef.co/go/tools/cmd/gosimple
```

Comment:
if go get fails, we should probably ignore the tests (go get from honnef fails pretty frequently. i do not really want this to block our entire CI). alternatively, we do not go get this every time.